src/App.jsx

```diff
@@ -1,25 +1,63 @@
-import { useState } from "react";
+import { useState, useEffect } from "react";
 import "./App.css";
 
-function City(props) {
-  return <div>This is the City component</div>;
+const api = "https://ctp-zip-code-api.onrender.com/";
+
+async function getZipCode(zipcode) {
+  const response = await fetch(api + "zip/" + zipcode);
```

Comment:
```suggestion
  const response = await fetch(`${api}zip/${zipcode}`);
```

Not that your way is wrong. Just this way may be a bit more readable and is more expected these days.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals